### PR TITLE
Improved helicopter special, rifle_milspec, and launcher spawns.

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Moved m1a spawn from helicopter itemgroup and replaced it with m14ebr
Moved all launchers and launcher ammo to guns_launcher_milspec and increased/rebalanced ticket amounts
~Added RPG-7 launcher and ammo to guns_launcher_milspec~
Added RPG-7 ammo to launchers
Added guns_launcher_milspec and guns_rifle_milspec to helicopter
Added seats, bloody rags, and human bones as Mecares suggested, and also added human flesh

### CODE REVIEW

**Comment by Mecares** [comment]
Maybe you could also add a chance for human bones and bloody rags to the helicopter crash and maybe seats.

File: data/json/itemgroups/guns.json [hunk 1]
```diff
@@ -309,6 +310,15 @@
     "id": "guns_launcher_rocket",
     "//": "Factory manufactured rocket launchers (excludes improvised weapons)",
     "items": [
+      [ "LAW_Packed", 200 ],
+      [ "RPG", 5 ],
+      [ "RPG-7_ammo", 10 ],
```
Comment (illi-kun) [inline data/json/itemgroups/guns.json]: This wasn't added here initially since it's weird to use russian weapon in the US by US army.

**Comment by BorkBorkGoesTheCode** [comment]
@Mecares I've done so.

File: data/json/item_groups.json [hunk 2]
```diff
@@ -6857,6 +6857,10 @@
     "items":[
       { "group": "guns_launcher_milspec", "prob": 10 },
       { "group": "guns_rifle_milspec", "prob": 10 },
+      { "item": "human_flesh", "prob": 4, "charges-min": 1, "charges-max": 10 },
+      { "item": "bone_human", "prob": 10 "charges-min": 1, "charges-max": 5 },
```
Comment (illi-kun) [inline data/json/item_groups.json]: Missed comma before charges min here and below.

Comment (BorkBorkGoesTheCode) [reply data/json/item_groups.json]: I saw that and fixed it with the seat spawn chance correction

File: data/json/item_groups.json [hunk 3]
```diff
@@ -6855,10 +6855,15 @@
     "id" : "helicopter",
     "magazine": 100,
     "items":[
+      { "group": "guns_launcher_milspec", "prob": 10 },
+      { "group": "guns_rifle_milspec", "prob": 10 },
+      { "item": "human_flesh", "prob": 4, "charges-min": 1, "charges-max": 10 },
+      { "item": "bone_human", "prob": 10, "charges-min": 1, "charges-max": 5 },
+      { "item": "rag_bloody", "prob": 10, "charges-min": 1, "charges-max": 40 },
```
Comment (illi-kun) [inline data/json/item_groups.json]: charges-min & charges-max mean something only for items with charges.
BTW, what ``` "magazine": 100,``` means for item group??!

Comment (BorkBorkGoesTheCode) [reply data/json/item_groups.json]: Magazine 100 means every gun always spawns with a magazine.

File: data/json/item_groups.json [hunk 4]
```diff
@@ -6855,10 +6855,15 @@
     "id" : "helicopter",
     "magazine": 100,
     "items":[
+      { "group": "guns_launcher_milspec", "prob": 10 },
+      { "group": "guns_rifle_milspec", "prob": 10 },
+      { "item": "human_flesh", "prob": 4, "count-min": 1, "count-max": 10 },
+      { "item": "bone_human", "prob": 10, "count-min": 1, "count-max": 5 },
+      { "item": "rag_bloody", "prob": 10, "count-min": 1, "count-max": 40 },
```
Comment (illi-kun) [inline data/json/item_groups.json]: 40 rags it's more than required for a blanket. Are you sure you want spawn that much rags?

**Comment by illi-kun** [comment]
Jenkins rebuild.

File: data/json/item_groups.json (hunk 4 above)
Comment (BorkBorkGoesTheCode) [reply data/json/item_groups.json]: Yes. The maximum spawn is still less than the total butcher yield of a full set of army clothes+armor x 8 (2 pilots+6 soldiers)

Comment (illi-kun) [reply data/json/item_groups.json]: Gathered in a single stack by someone? You can move it in a separate item group and spawn in smaller bunches, 40 rags in one stack is really weird.

Comment (BorkBorkGoesTheCode) [reply data/json/item_groups.json]: Would reducing the count by 1/4th be okay?

Comment (BorkBorkGoesTheCode) [reply data/json/item_groups.json]: Would reducing the count to 1/4th be okay?

Comment (illi-kun) [reply data/json/item_groups.json]: Yes.

Comment (BorkBorkGoesTheCode) [reply data/json/item_groups.json]: Done. Do I need to make any other changes?

**Comment by BorkBorkGoesTheCode** [comment]
Jenkins, rebuild please.

**Comment by BorkBorkGoesTheCode** [comment]
Jenkins, rebuild please.

**Comment by Mecares** [comment]
It would also be awesome if you could add sheet metal to the spawn, since it is a plausible item to spawn in a wreckage, uncraftable and almost impossible to find in wilderness scenario.

**Pull request merged**